packages/devtools_app/lib/src/flutter/status_line.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../devtools.dart' as devtools;
+
+/// The status line widget displayed at the bottom of DevTools.
+class StatusLine extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    // TODO(devoncarew): Break this into an isolates area, a connection status
+    // area, a page specific area, and a documentation link area.
+
+    final textTheme = Theme.of(context).textTheme;
+
+    return Container(
+      height: 24.0,
+      child: Center(
```

Comment:
instead of using center, set the alignment as part of the container.
@pq this an example of a use case for the lints to nudge users to use Container.

Reply:
Phil, can we collect ideas like this (this one, and the one below) in some central location? An issue tracker, or a doc?

Reply:
(changed to use the container's alignment)